snd-server/mappers/soundations.py

```diff
@@ -6,6 +6,7 @@ def create_track_from_spotify(track: SpotifyApiTrack) -> SoundationsTrack:
     return SoundationsTrack(
         id=track.id,
         name=track.name,
+        popularity=track.popularity,
```

Comment:
Unless this is used on the client, perhaps no need to add this field to `SoundationsTrack` though feel free to keep it :)